packages/forms-web-app/src/routes/home.js

```diff
@@ -1,9 +1,11 @@
 const express = require('express');
 const indexController = require('../controllers');
+const cookiePreferencesController = require('../controllers/cookies/cookie-preferences');
 
 const router = express.Router();
 
 /* GET home page. */
 router.get('/', indexController.getIndex);
+router.get('/cookies', cookiePreferencesController.getCookiePreferences);
```

Comment:
Imo, this would be better placed inside `routes/cookies.js`

https://github.com/Planning-Inspectorate/appeal-planning-decision/tree/master/packages/forms-web-app/tests/unit/routes

I'd create the following:

```
touch src/routes/cookies.js
touch tests/unit/routes/cookies/cookie.test.js
```

Where `src/routes/cookies.js`:

```
const express = require('express');
const cookiePreferencesController = require('../controllers/cookies/cookie-preferences');

const router = express.Router();

router.get('/cookies', cookiePreferencesController.getCookiePreferences);

module.exports = router;
```

And `tests/unit/routes/cookies/cookie.test.js`:

```
const { get } = require('./router-mock');
const cookiePreferencesController = require('../../../src/controllers/cookie-preferences');

describe('routes/index', () => {
  beforeEach(() => {
    // eslint-disable-next-line global-require
    require('../../../src/routes/cookies');
  });

  afterEach(() => {
    jest.resetAllMocks();
  });

  it('should define the expected routes', () => {
    expect(get).toHaveBeenCalledWith('/', cookiePreferencesController.getCookiePreferences);
  });
});

```

There would be a test to update as well - https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/master/packages/forms-web-app/tests/unit/routes/index.test.js

```
   const cookieRouter = require('../../../src/routes/cookies');

// ...

    expect(use).toHaveBeenCalledWith('/cookies', cookieRouter);
    expect(use.mock.calls.length).toBe(4);
```

Reply:
Rest looks good to me 👍